core/include/lifo.h

```diff
@@ -23,32 +24,47 @@
 #ifndef LIFO_H
 #define LIFO_H
 
+#define ENABLE_DEBUG 0
+#include "debug.h"
```

Comment:
Please no `DEBUG` in headers. This will lead to debug being enabled whenever the file is included. If you want to keep the `DEBUG` statements, use `LOG_DEBUG` instead.

Reply:
will change